client/src/modules/list/model/reducer.js

```diff
@@ -12,6 +12,14 @@ import items from 'modules/list/components/Items/model/reducer';
 import { ListType } from 'modules/list/consts';
 import { filterDefined } from 'common/utils/helpers';
 import { CommonActionTypes } from 'common/model/actionTypes';
+import { DISPLAY_LIMIT } from 'modules/list/components/Items/ItemsList';
+
+const listState = {
+  archivedLimit: DISPLAY_LIMIT,
+  areArchivedItemsDisplayed: false,
+  doneLimit: DISPLAY_LIMIT,
+  unhandledLimit: DISPLAY_LIMIT
```

Comment:
do we need three properties that represent the same one thing??
can't we just use one property of `itemsLimit`  ??

```
const listState = {
  areArchivedItemsDisplayed: false,
  itemsLimit: DISPLAY_LIMIT
```

Reply:
on the list view, we have 3 "sublists" (itemsLists) we display separately done items unhandled items and archived items. Each `itemsLists` has its own limit (DISPLAY_LIMIT) for displayed items.